testing/tests/install-ignore-dirty

```diff
@@ -0,0 +1,16 @@
+# Test invalid package names and paths
+# @TEST-EXEC: bash %INPUT 2>output
+# @TEST-EXEC: btest-diff output
```

Comment:
Just making sure, do we need to check that the output is empty on top of being able to run from a dirty directory? To me it seems `output` is secondary and we could just drop it here.

Reply:
I removed the output but added a couple of `test`s to make sure the output files from install and bundle exist correctly.